go.mod

```diff
@@ -130,6 +129,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require golang.org/x/exp v0.0.0-20220823124025-807a23277127
```

Comment:
how come this is separate?

Reply:
no idea, merged it into the main `require` block